deploy/006-deploy-swaprouter.ts

```diff
@@ -1,27 +1,36 @@
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
-import ADDRESSES from "../helpers/address";
+// MAINNET DEPLOYED CONTRACTS
+import Mainnet from "../deployments/bscmainnet.json";
+// TESTNET DEPLOYED CONTRACTS
+import Testnet from "../deployments/bsctestnet.json";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, network, getNamedAccounts } = hre;
+  const { deployments, getNamedAccounts } = hre;
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
 
-  const networkName = network.name === "bscmainnet" ? "bscmainnet" : "bsctestnet";
-  const addresses = ADDRESSES[networkName];
-  const WBNBAddress = addresses.wbnb;
-  const pancakeFactoryAddress = addresses.pancakeFactory;
+  const unitrollerAddresses = (await deployments.get("Unitroller")).address;
+  const wBNBAddress = (await deployments.get("WBNB")).address;
+  const vbnbAddress = (await deployments.get("vBNB")).address;
+  // Pancake Factory doesn't exist on hardhat so we are using the testnet address
+  const pancakeFactoryAddress =
+    hre.network.name === "bscmainnet"
+      ? Mainnet.contracts.pancakeFactory.address
+      : Testnet.contracts.pancakeFactory.address;
```

Comment:
Could we use `(await deployments.get("pancakeFactory")).address`?

Reply:
We can't here because it will fail locally. So we'd need to deploy this on hardhat to remove the "faked" address on the hardhat network